front/app/components/admin/PostManager/components/FilterSidebar/FilterSidebarStatuses.tsx

```diff
@@ -56,30 +56,41 @@ const FilterSidebarStatuses = ({
           <FormattedMessage {...messages.allStatuses} />
         </Menu.Item>
         <Divider />
-        {/* Only input statuses can be edited and only admins can do this */}
+        {/* Input statuses can be edited and only admins can do this */}
         {isAdmin(authUser) &&
-          (type === 'AllIdeas' ||
-            (type === 'ProjectIdeas' && (
-              <Box display="inline-flex">
-                <Button
-                  buttonStyle="text"
-                  icon="edit"
-                  pl="12px"
-                  linkTo="/admin/settings/ideation/statuses"
-                  iconPos="right"
-                  iconSize="14px"
-                >
-                  <Text
-                    m="0px"
-                    color="coolGrey600"
-                    fontSize="s"
-                    textAlign="left"
-                  >
-                    <FormattedMessage {...messages.editStatuses} />
-                  </Text>
-                </Button>
-              </Box>
-            )))}
+          (type === 'AllIdeas' || type === 'ProjectIdeas') && (
+            <Box display="inline-flex">
```

Comment:
The code in this box and the one for proposals looks pretty much identical, except that the `linkTo` is different. Could it have been possible to use a variable to keep the right value for `linkTo` and only have the code for the box once? Or perhaps this is not ideal when using JSX?

Reply:
Good point! I changed it.